pkgs/data/fonts/lexend/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  version = "0.pre+date=2022-01-27";
+  pname = "lexend";
```

```suggestion
  pname = "lexend";
  version = "0.pre+date=2022-01-27";
```
pname typically goes first